frontend/pages/user/edit-profile.js

```diff
@@ -10,6 +10,10 @@ import withPrivateRoute from '../../components/utils/withPrivateRoute'
 function EditProfile (props) {
   const { user } = props
 
+  if (!user) {
+    return <div/>
```

Comment:
```suggestion
    return null
```
wouldn't it be better?